kubetest/azure.go

```diff
@@ -537,7 +537,36 @@ func (c Cluster) Down() error {
 }
 
 func (c Cluster) DumpClusterLogs(localPath, gcsPath string) error {
-	return nil
+	// should run on master node and collect logs through winrm from windows slaves
+
+	const AzureLogDumpTemplate = `
+function wrapper-ssh () {
+	ssh -i %[1]s -o StrictHostKeyChecking=no "$@"
+}
+function wrapper-scp () {
+	scp -i %[1]s -o StrictHostKeyChecking=no "$@"
+}
+wget https://raw.githubusercontent.com/e2e-win/e2e-win-prow-deployment/master/custom-scripts/master.sh
```

Comment:
I wouldn't have this dependency here. Why not just use the masterscript as a template too ?